### PR TITLE
Move `getUiEventSeries` into common

### PR DESCRIPTION
**Pull request by davidpmccormick**
⬆️

### CODE REVIEW

File: common/services/prismic/events.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {getDocument} from './api';
+import {parseBackgroundTexture} from './parsers';
+import type {UiEventSeries} from '../../model/events';
+
+export async function getUiEventSeries(req: Request, id: string): Promise<UiEventSeries> {
+  const series = await getDocument(req, id, {
+    fetchLinks: ['background-textures.image', 'background-textures.name']
```
Comment (jamesgorrie) [inline common/services/prismic/events.js]: Could we stick this in `fetchLinks` for consistency?

File: common/services/prismic/events.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  });
+
+  const prismicBackgroundTexture = series && series.data && series.data.backgroundTexture && series.data.backgroundTexture.data;
```
Comment (jamesgorrie) [inline common/services/prismic/events.js]: When would you ever have series and not series .data as it's part of `PrismicDocument`?

File: common/services/prismic/events.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  return {
+    id: id,
+    title: series && series.data && series.data.title || 'TITLE MISSING',
+    description: series && series.data && series.data.description,
```
Comment (jamesgorrie) [inline common/services/prismic/events.js]: https://github.com/wellcometrust/wellcomecollection.org/pull/2442/files#r179421821

File: common/model/background-texture.js [hunk 4]
```diff
@@ -1,5 +1,12 @@
 // @flow
 
+export type PrismicBackgroundTexture = {|
+  image: {
+    url: string
+  },
+  name: string
+|}
```
Comment (jamesgorrie) [inline common/model/background-texture.js]: We haven't been doing this for other things, we're just using PrismicFragment - do you think we should be doing this?

Comment (davidpmccormick) [reply common/model/background-texture.js]: Ah yeah, this is maybe too paranoid. I'll use PrismicFragment instead.

File: common/services/prismic/events.js (hunk 2 above)
Comment (davidpmccormick) [reply common/services/prismic/events.js]: Good point – I wasn't using `getDocument` before the move to common, and wasn't checking for a `PrismicDocument` type. I'll update.

**Pull request merged**